packages/help-center/src/components/help-center-support-chat-message.tsx

```diff
@@ -86,6 +88,8 @@ export const HelpCenterSupportChatMessage = ( {
 		);
 	};
 
+	const hasUnread = unreadMessages > 0;
```

Comment:
Tangentially related to my previous message, and certainly a bit nitpicky but something you may want to consider:
```suggestion
	const hasUnreadMessages = unreadMessages > 0;
```